src/components/GDPRConsent.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, { useEffect } from 'react';
+
+import { GTM_ID } from '../constants';
+import CookieConsent, { getCookieConsentValue }  from "react-cookie-consent";
+import TagManager from 'react-gtm-module';
+
+
+const GDPRConsent = () => {
+  
+  const onAccept = () => {
+    // Nothing yet
+  };
+  
+  useEffect(() => {
+    // Just to ensure that when we initialize GTM, this message will be shown
+    TagManager.initialize({ gtmId: GTM_ID });
+
+    const isConsent = getCookieConsentValue();
+    if (isConsent === "true") {
+      onAccept();
```

Comment:
What is it exactly doing since the `onAccept()` method does nothing?

Reply:
In case of we decide to put something that will run only if the user accept. A "consent cookie" is saved and next time the user access the site, we don't need to ask again, the consent is stored. So, we have to ensure the same behavior as he click on accept,

Reply:
I thought the component would already do it. If it does, I think we must do it exactly as you described.

Reply:
I explain it here: https://github.com/HathorNetwork/internal-issues/issues/29

Reply:
The way it is developed, will this message appear every refresh?